qulice-pmd/src/main/resources/com/qulice/pmd/ruleset.xml

```diff
@@ -196,4 +198,17 @@
         </properties>
         <priority>3</priority>
     </rule>
+    <rule name="AvoidDuplicateLiterals"
+          message="The String literal {0} appears {1} times in this file; the first occurrence is on line {2}"
+          class="net.sourceforge.pmd.lang.java.rule.strings.AvoidDuplicateLiteralsRule">
+        <description>
+            Code containing duplicate String literals can usually be improved by declaring the String as a constant field.
+        </description>
+        <priority>3</priority>
+        <properties>
+            <property name="maxDuplicateLiterals">
```

Comment:
@dskalenko I think it's enough to just modify this property, no need to redefine the check